platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/turtle/TurtleTimeManager.java

```diff
@@ -94,6 +94,10 @@ public void addTimeTickReceiver(@NonNull final TimeTickReceiver receiver) {
         timeTickReceivers.add(receiver);
     }
 
+    public void clear() {
+        timeTickReceivers.clear();
```

Comment:
Is this really needed? The `TurtleTimeManager` should be destroyed together with the `TestEnvironment`. I don't think we need to remove the observers.

Reply:
Removed